audit/src/main/java/gov/cms/ab2d/audit/cleanup/FileDeletionServiceImpl.java

```diff
@@ -133,7 +218,7 @@ private boolean deleteFile(Path path) {
                     logFileNotEligibleForDeletion(path);
                 }
             }
-            // Actually log here because if we've gotten here with an exception, we deleted it.
+            // Actually log here because if we've gotten here without an exception, we deleted it.
```

Comment:
That's actually not true.  Looking at execution paths, there are 2 logFileNotEligibleForDeletion() calls that could have occurred.

To fix: refactor to method lines 205 - 219 and move current lines 221 - 224 into that method.

The fileEvent variable will now be completely contained within that method.

Reply:
This is addressed by the refactor I did.